src/buffer-builder.ts

```diff
@@ -182,6 +182,18 @@ export class BufferBuilder {
     return this;
   }
 
+  /**
+   * Register open cash drawer action
+   * @return BufferBuilder
+   */
+  public openCashDrawer(): BufferBuilder {
+    // kick drawer 1, pin 2
```

Comment:
I think we can listen to drawer status too!

Reply:
i dont recall exact status it will be something like `ALB`!

Reply:
I don't get it.
This command sends a pulse to cash drawer.

Reply:
You want it not to send pulse if it is already opened?